galaxyproject/galaxy
# Add Toolbox Filters Button to Toolbox

fixes #10905 by adding a button with the tool-tip "Toolbox Filter Settings" to the toolbox, which navigates to said setting in the user preferences.

![image](https://user-images.githubusercontent.com/44241786/183103964-e8ecd82e-33cc-4466-883d-8c22ff94e34d.png)

This makes more users aware of the feature by advertising it right next to where it takes effect, and acts as a convenient shortcut to access the filter settings.

## How to test the changes?
  1. Log in
  2. Press the filter button in the toolbox
  3. The toolbox filter settings should open

## License
